### PR TITLE
[3664834472] Add modules version to analytics

### PR DESCRIPTION
**Pull request by rYuuk**
#### Added

- Version to ModuleInfo
- Sending of all modules version to amplitude

#### Updated

- Fix PartnerdomainSettings capitalization

## How to Test

- Import the modules in empty project and check amplitude events

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

### CODE REVIEW

File: Editor/Module Management/ModuleInstaller.cs [hunk 1]
```diff
@@ -135,5 +137,27 @@ private static void AppendScriptingSymbol()
...
+            {
+                if (packageList.All(x => x.name != module.name))
+                {
+                    allModuleInstalled = false;
+                }
+            }
+           
+            if (allModuleInstalled)
+            {
+                SDKLogger.Log(TAG, "All the modules are installed successfully. Ready Player Me avatar system is ready to use.");
```
Comment (HarrisonHough) [inline Editor/Module Management/ModuleInstaller.cs:155]: These hard-coded log messages could be moved to a const. 
minor thing though :D

**Review by HarrisonHough — APPROVED** [review]
Cool seems to work. I assume you will do a separate PR for logging Avatar Loader version too?

**Comment by srcnalt** [comment]
I am getting quite many merge conflicts and errors in Unity, @rYuuk can you take a look? This happens in the Unity-SDK project.

**Comment by rYuuk** [comment]
We need to merge this PR first https://github.com/readyplayerme/Unity-Avatar-Loader/pull/31 as the GLTFast has been updated to v5 and @richieRPM haven't looked at this one.

**Review by richieRPM — APPROVED** [review]
gltfast 5... lets go!

**Comment by richieRPM** [comment]
> 

Need to sort out all the PRs you guys need approved now... its ridiculous... no link between them all either

**Pull request merged**